client/python/tensorflow-serving-api/samples/README.md

```diff
@@ -28,7 +28,8 @@ pip3 install -r requirements.txt
 Access to Google Cloud Storage might require proper configuration of https_proxy in the docker engine or in the docker container.
 In the examples listed below, OVMS can be started using a command:
 ```bash
-docker run -d --rm -e "http_proxy=$http_proxy" -e "https_proxy=$https_proxy" -p 8000:8000 -p 9000:9000 openvino/model_server:latest --model_name resnet --model_path gs://ovms-public-eu/resnet50 --port 9000 --rest_port 8000
+wget https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.{xml,bin} -P models/resnet50/1
+docker run -d -u $(id -u) -v $(pwd)/models:/models -p 8000:8000 -p 9000:9000 openvino/model_server:latest --model_name resnet --model_path /models/resnet50 --port 9000 --rest_port 8000
 ```
```

Comment:
Test fails due to:
 File "grpc_predict_resnet.py", line 133, in <module>
E               result = stub.Predict(request, 10.0) # result includes a dictionary with all model outputs
E             File "/tmp/pytest-of-k8sworker/pytest-566/test_commands_ovms_client_pyth0/model_server/client/python/tensorflow-serving-api/samples/.venv/lib/python3.8/site-packages/grpc/_channel.py", line 946, in __call__
E               return _end_unary_response_blocking(state, call, False, None)
E             File "/tmp/pytest-of-k8sworker/pytest-566/test_commands_ovms_client_pyth0/model_server/client/python/tensorflow-serving-api/samples/.venv/lib/python3.8/site-packages/grpc/_channel.py", line 849, in _end_unary_response_blocking
E               raise _InactiveRpcError(state)
E           grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
E           	status = StatusCode.INVALID_ARGUMENT
E           	details = "Missing input with specific name - Required input: 0"
E           	debug_error_string = "UNKNOWN:Error received from peer ipv6:%5B::1%5D:31666 {grpc_message:"Missing input with specific name - Required input: 0", grpc_status:3, created_time:"2023-01-13T13:49:26.190459032+01:00"}"

Reply:
Output errors:

command:
1) python grpc_get_model_metadata.py --grpc_port 31666 --model_name resnet --model_version 1: 
a) Input name: data; shape: [1, 3, 224, 224]; dtype: DT_FLOAT -> Input name: 0; shape: [1, 3, 224, 224]; dtype: DT_FLOAT
b) Output name: prob; shape: [1, 1000]; dtype: DT_FLOAT -> Output name: 1463; shape: [1, 1000]; dtype: DT_FLOAT
2) python rest_get_model_metadata.py --rest_port 31000: 
a)  "data": { -> "0": {
b) "name": "data" -> "name": "0"
c)  "prob": { -> "1463": {
d) "name": "prob" -> "name": "1463"